utils/wasm-proc/src/main.rs

```diff
@@ -188,15 +188,14 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
 
         let file = PathBuf::from(file);
-        // Issue (#1971)
```

Comment:
Solve this one also pls
https://github.com/gear-tech/gear/blob/ec3d91fc33c445104922d5290e1302493819acff/utils/wasm-builder/src/wasm_project.rs#L240-L241